src/main.c

```diff
@@ -177,7 +177,8 @@ main(int argc, char **argv)
         return 0;
     }
 
-    prof_run(log ? log : "INFO", account_name, config_file, log_file, theme_name);
+    /* Default logging WARN */
+    prof_run(log ? log : "WARN", account_name, config_file, log_file, theme_name);
```

Comment:
so we only log warnings to the logfile, seems fine.

Reply:
Yes, and in the second round of clean-up we can add some helpful information with "info" for users and developer, which can be enabled very easy.